src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+  setSelectedTodo: (todo: Todo) => void;
+  selectedTodo: Todo | null;
+};
+
+export const TodoInfo: React.FC<Props> = ({
+  todo,
+  setSelectedTodo,
+  selectedTodo,
+}) => {
+  const isSelected = selectedTodo ? selectedTodo.id === todo.id : false;
```

Comment:
Why did you use ternary here?

```suggestion
  const isSelected = selectedTodo.id === todo.id;
```

Reply:
because, it's possibly null, so I'd used ternary by inertia
changed it to: 
`const isSelected = selectedTodo?.id === todo.id;`